src/NuGet.Services.Metadata.Catalog.Monitoring/Resources/PackageRegistrationMetadataResourceDatabaseFeed.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Packaging.Core;
+using NuGet.Services.Metadata.Catalog.Helpers;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class PackageRegistrationMetadataResourceDatabase : IPackageRegistrationMetadataResource
```

Comment:
This class was renamed from `PackageRegistrationMetadataResourceV2Feed` and had its code changed to pull data from `IGalleryDatabaseQueryService` instead of `V2FeedParser`.